src/Stratis.Bitcoin.Features.Notifications.Tests/BlockNotificationTest.cs

```diff
@@ -34,7 +36,7 @@ public BlockNotificationTest()
         /// Tests that <see cref="BlockNotification.Notify(System.Threading.CancellationToken)"/> exits due
         /// to <see cref="BlockNotification.StartHash"/> being null and no blocks were signaled.
         /// </summary>
-        [Fact]
+        [Fact(Skip = SkipReason)]
```

Comment:
wouldn't it be better if the text right here rather than a const ?   It is the same text so a const is valid, but since you would want to see the reason, it makes sense have it right next to it here.

Compromise would be to name the const   RevisitWhenBlockNotificationFixed